packages/react-components/src/components/chart/contextMenu/ChartContextMenu.spec.tsx

```diff
@@ -0,0 +1,20 @@
+import { describe, expect } from '@jest/globals';
+import { render } from '@testing-library/react';
+import React from 'react';
+import ChartContextMenu from './ChartContextMenu';
+
+describe('ChartContextMenu', () => {
+  const menuOptionClickHandler = jest.fn();
+  const onOutSideClickHandler = jest.fn();
+  const component = render(
+    <ChartContextMenu
+      menuOptionClickHandler={menuOptionClickHandler}
+      onOutSideClickHandler={onOutSideClickHandler}
+      position={{ x: 100, y: 100 }}
+      trendCursors={[]}
+    />
+  );
+  it('renders', () => {
+    expect(component).not.toBeNull();
+  });
```

Comment:
Nothing more we should test, such as the existence of each action? Where would we catch the issue if one of the context menu actions was disable on accident?

Reply:
I am looking into how can we trigger events on charts, havent been successful so far.